src/hooks/useArticle.js

```diff
@@ -12,17 +12,23 @@ export const useArticle = (lang, title) => {
     Promise.all([getArticle(lang, title), getArticleMedia(lang, title), getSuggestedArticles(lang, title)])
       .then(([article, media, suggestedArticles]) => {
         const { sections, toc } = article
+        const i18nLocale = i18n.locale

```

Comment:
Is this still needed here?

Reply:
Even though it seems a bit redundant to set and reset `locale` here after commit b0261c3f403a003c96e1170c380da4997ec7b217, we still need to do the brief `locale` switch in `useArticle` if we want the "Suggested articles" label to show in the article language in the section menu popup (becuase `sectionsWithFooter` and `tocWithFooter` are constructed here)

<img width="276" alt="image" src="https://user-images.githubusercontent.com/4752599/76777436-931e5400-677e-11ea-8d64-4749808e13ff.png">

So it's a bit of a trade-off that comes from simplifying `Article.js` in commit b0261c3f403a003c96e1170c380da4997ec7b217. What do you think, is that commit/trade-off worth it?

Reply:
I think we can leave it here, in the long run, we should think of how to refactor to better use of i18n to separate app language and content language.

Currently, it is super unclear whether the lang is for app or content lang. The lang here is for content, but the other places are for app which confuse.